initfiles/componentfiles/config2xml/esp.xsd

```diff
@@ -79,40 +79,47 @@
                             </xs:complexType>
```

Comment:
need a space between use="optional" and hpcc:displayName=

Reply:
Changed. Incidentally, the lack of a space does not affect the parsing.